### ISSUE TITLE
Activity Zone 'zone becomes occupied' trigger not firing

### ISSUE THREAD

**Issue by rbdubz3** [FlyingDiver/Indigo-Occupatum#13]
I recently integrated a _'zone becomes occupied'_ trigger into my setup. The 'Activity Zone' will go into the ON/OFF states properly, however I do not see the trigger event firing

**Comment by FlyingDiver** [comment]
Honestly, I'm not even sure why I put those events in there, since they should be identical to just triggering on the state becoming "on" or "off".

**Comment by FlyingDiver** [comment]
In any case, try this one before I push it to the store:  https://github.com/FlyingDiver/Indigo-Occupatum/releases/tag/2022.0.5

**Comment by rbdubz3** [comment]
Yeah - no worries Joe - i did just take that route and setup a trigger as general 'Device ON' and it is all fine.. So if you'd rather just remove those specific triggers and use the native indigo devices ones that seems fine as well ..

**Comment by FlyingDiver** [comment]
Well, I don't like removing existing functionality, so I won't remove the triggers.  Can you test that the triggers work correctly with Activity Zones?

**Comment by rbdubz3** [comment]
giving it a shot - i'm getting some plugin startup errors with that tag release 2022.0.5 :

```

   Starting plugin "Occupatum 2022.0.5" (pid 17951)
   Occupatum Debug                 logLevel = 10
   Started plugin "Occupatum 2022.0.5"
   Occupatum                       Starting Occupatum
   Occupatum                       Avas Bedroom Activity Zone: Starting Device
   Occupatum Debug                 Avas Bedroom Activity Zone: Zone 1217906877 uses sensor devices: [174822628]
   Occupatum Debug                 Avas Bedroom Activity Zone: watchList updated: {174822628: [1217906877]}
   Occupatum Error                 exception in device_start_comm(Avas Bedroom Activity Zone): name 'device' is not defined
   Occupatum                       Cocos Bedroom Activity Zone: Starting Device
   Occupatum Debug                 Cocos Bedroom Activity Zone: Zone 1347465739 uses sensor devices: [146711811]
   Occupatum Debug                 Cocos Bedroom Activity Zone: watchList updated: {174822628: [1217906877], 146711811: [1347465739]}
   Occupatum Error                 exception in device_start_comm(Cocos Bedroom Activity Zone): name 'device' is not defined
   Occupatum                       Great Room Activity Zone: Starting Device
   Occupatum Debug                 Great Room Activity Zone: Zone 734905279 uses sensor devices: [523578307, 631803371, 386136298]
   Occupatum Debug                 Great Room Activity Zone: watchList updated: {174822628: [1217906877], 146711811: [1347465739], 523578307: [734905279], 631803371: [734905279], 386136298: [734905279]}
   Occupatum Error                 exception in device_start_comm(Great Room Activity Zone): name 'device' is not defined
   Occupatum                       Master Bedroom Activity Zone: Starting Device
   Occupatum Debug                 Master Bedroom Activity Zone: Zone 73060101 uses sensor devices: [1348516012, 514451044, 1807471232]
   Occupatum Debug                 Master Bedroom Activity Zone: watchList updated: {174822628: [1217906877], 146711811: [1347465739], 523578307: [734905279], 631803371: [734905279], 386136298: [734905279], 1348516012: [73060101], 514451044: [73060101], 1807471232: [73060101]}
   Occupatum Error                 exception in device_start_comm(Master Bedroom Activity Zone): name 'device' is not defined
   Occupatum                       Media Room Activity Zone: Starting Device
   Occupatum Debug                 Media Room Activity Zone: Zone 116605528 uses sensor devices: [1132600242, 755959762]
   Occupatum Debug                 Media Room Activity Zone: watchList updated: {174822628: [1217906877], 146711811: [1347465739], 523578307: [734905279], 631803371: [734905279], 386136298: [734905279], 1348516012: [73060101], 514451044: [73060101], 1807471232: [73060101], 1132600242: [116605528], 755959762: [116605528]}
   Occupatum Error                 exception in device_start_comm(Media Room Activity Zone): name 'device' is not defined
```

**Comment by FlyingDiver** [comment]
Grrr.  OK, I'll get back to that one.

**Comment by FlyingDiver** [comment]
Would you mind testing this again:  https://github.com/FlyingDiver/Indigo-Occupatum/releases/tag/2022.0.5

Same version number so you'll have to install manually.

**Comment by rbdubz3** [comment]
Ok - so far so good - new tag version seems better and successfully fires the 'zone becomes occupied' trigger..

thanks Joe!

**Comment by FlyingDiver** [comment]
Fixed in 2022.0.5
